datahub-frontend/app/controllers/Application.java

```diff
@@ -170,6 +170,7 @@ private ObjectNode userEntityProps() {
    */
   @Nonnull
 
+  // What is this good for??
```

Comment:
This comment will probably only add more confusion. Please open a ticket asking about this instead.

My guess is this is code that was exported from internal code.

Reply:
This shouldn't have been in here... will remove